packages/appstore/src/components/onboarding-new/static-cfd-account-manager.tsx

```diff
@@ -201,7 +203,7 @@ const StaticCFDAccountManager = ({
                             primary
                             className={classNames('static-cfd-account-manager__buttons-trade', {
                                 'static-cfd-account-manager__buttons-trade--blurry': is_blurry.trade,
-                                'static-cfd-account-manager__buttons-topup--animated': is_onboarding_animated.trade,
+                                'static-cfd-account-manager__buttons-trade--animated': is_onboarding_animated.trade,
```

Comment:
@arshad-rao-deriv why do we need to change the class of this button?

Reply:
@mahdiyeh-deriv it's because in the code we use `static-cfd-account-manager__buttons-TOPUP--animated` class for top-up button and for trade button too, but `static-cfd-account-manager__buttons-TRADE--animated` class we didn't use anywhere, it existed but never used before, so @arshad-rao-deriv just fixed this misspelling